tests/register.spec.coffee

```diff
@@ -29,10 +27,10 @@ describe 'Device Register:', ->
 
 	describe '.register()', ->
 
-		beforeEach ->
+		before ->
 			mockServer.start()
 
-		beforeEach ->
+		before ->
```

Comment:
`mockServer.start()` could be in the same `before`

Reply:
Had that in mind, but preferred a second `before` instead of a `.then` to keep the code a bit shallower.